react-debugging.html

```diff
@@ -26,9 +24,10 @@
           setMood: null
         }
       }
-
       nextMood = () => {
-        if(currentMood === moods.length - 1){
+        let currentMood = 0
+        //was getting an error saying that currentMood was undefined so I declared it above, then it said moods was undefined. so I added this.state, I got it to work but it only works for one click. Afterwards clicking it again doesn't do anything and it doesn't give me errors on the browser console. At this point I'm kind of stuck.
```

Comment:
So this is where I think why your version of the code does not loop through back on itself, since you are declaring "let currentMood = 0" its no longer referring to the state of mood, but rather the variable it's current at.